workloads/README.md

```diff
@@ -0,0 +1,64 @@
+## Deploying AgnosticD workloads
+
+This is a collection of scripts that help you deploy workloads on AgnosticD OpenShift clusters.
+
+These AgnosticD workloads assist in automated deployment of:
+ - The OpenShift 3->4 Application Migration Tool 
+ - Various sample apps for use when testing App Migration capabilities from OpenShift 3->4  
+
+### NOTE
+
+All workloads deployed by mig-agnosticd must be present in the main AgnosticD directory on your machine. However, the workloads we'll use in this README are yet not part of the redhat-cop/agnosticd repo (see PR 489). To use these workloads before PR 489 merges, clone the following repo and set `AGNOSTICD_HOME` to the clone path:
+
+```bash
+git clone https://github.com/pranavgaikwad/agnosticd
+```
+
+### Deploying a workload
+
+To create a new workload:
+
```

Comment:
Suggest we add a little "usage" section for the shell script.

#### Usage: `./deploy_workload.sh `
|Flag|Purpose|Accepted Values|
|---|---|---|
|`-a`|Workload Action|[`create`, `delete`]|
|`-w`|Workload Name|[`migration`, `mssql`, ...]|
|`-v`|AgnosticD OpenShift Version|[`3`, `4`]|

```
#### Usage: `./deploy_workload.sh `
|Flag|Purpose|Accepted Values|
|---|---|---|
|`-a`|Workload Action|[`create`, `delete`]|
|`-w`|Workload Name|[`migration`, `mssql`, ...]|
|`-v`|AgnosticD OpenShift Version|[`3`, `4`]|
```